neo4j-store/src/main/kotlin/org/ostelco/prime/storage/graph/Neo4jStore.kt

```diff
@@ -1813,7 +1828,9 @@ object Neo4jStoreSingleton : GraphStore {
                                 data = extendedStatus
                         )
                     }
-                    logger.info(NOTIFY_OPS_MARKER, "Jumio verification failed for ${customer.contactEmail} Info: $extendedStatus")
+                    if(updatedScanInformation.scanResult?.verificationStatus != "NO_ID_UPLOADED") {
```

Comment:
Also, for the failed case, not logging for NO_ID_UPLOADED (when Customer cancels KYC).